CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Onboarding/Views/OnboardingPageCell.swift

```diff
@@ -15,6 +15,14 @@ class OnboardingPageCell: UICollectionViewCell {
 
     override func awakeFromNib() {
         super.awakeFromNib()
+        title.textColor = GiniColor(
+            light: UIColor.GiniCapture.accent1,
```

Comment:
[Change request]: 
<img width="189" alt="Screenshot 2022-11-02 at 18 23 07" src="https://user-images.githubusercontent.com/3911816/199558897-5d8f638a-a678-443c-b0d9-cabe106b0999.png">
The title color is not UIColor.GiniCapture.accent1 - it should be black (from the palette)